src/shell/command_helper.h

```diff
@@ -499,36 +501,40 @@ inline void scan_data_next(scan_data_context *context)
                             context->timeout_ms);
                         break;
                     case SCAN_COUNT:
-                        context->split_rows++;
-                        if (context->stat_size && context->statistics) {
-                            long hash_key_size = hash_key.size();
-                            context->statistics->measureTime(
-                                static_cast<uint32_t>(histogram_type::HASH_KEY_SIZE),
-                                hash_key_size);
-
-                            long sort_key_size = sort_key.size();
-                            context->statistics->measureTime(
-                                static_cast<uint32_t>(histogram_type::SORT_KEY_SIZE),
-                                sort_key_size);
-
-                            long value_size = value.size();
-                            context->statistics->measureTime(
-                                static_cast<uint32_t>(histogram_type::VALUE_SIZE), value_size);
-
-                            long row_size = hash_key_size + sort_key_size + value_size;
-                            context->statistics->measureTime(
-                                static_cast<uint32_t>(histogram_type::ROW_SIZE), row_size);
-
-                            if (context->top_count > 0) {
-                                context->top_rows.push(
-                                    std::move(hash_key), std::move(sort_key), row_size);
+                        if (kv_count == -1) {
```

Comment:
Does `kv_count == -1` means `only_return_count` ? Can `only_return_count` be used as the condition to be checked ?

Reply:
In order to cope with the different versions of the match, I think this is the best